package.json

```diff
@@ -57,6 +57,7 @@
     "react": ">=16.8.3",
     "react-native": ">=0.59.0",
     "react-native-navigation": ">=6.5.0",
+    "@types/react-native-navigation": "2.0.0",
```

Comment:
Do we really need to add this dependency? 

AFAIK **react-native-navigation** provide the types as part of the library so you don't need to add the types from and external package.

![image](https://user-images.githubusercontent.com/95109/82240113-40911d80-9910-11ea-8ed1-3c24139c3a47.png)